uint8 ACTUATOR_CMD=0
uint8 CONTROL_GROUP_CMD=1
uint8 ATTITUDE_RATE_CMD=2
uint8 ATTITUDE_CMD=3
uint8 ACCELERATION_CMD=4
uint8 VELOCITY_CMD=5
uint8 POSITION_CMD=6

uint8 input_type

bool armed
